tests/distributions/test_distribution.py

```diff
@@ -83,6 +83,15 @@ def test_issue_4499(self):
             x = pm.DiracDelta("x", 1, size=10)
         npt.assert_almost_equal(m.compile_logp()({"x": np.ones(10)}), 0 * 10)
 
+    def test_issue_7548(self):
```

Comment:
Give test an informative name, and it wasn't really a bug but missing functionality